Userland/Applications/Debugger/main.cpp

```diff
@@ -11,15 +11,20 @@
 #include <AK/Platform.h>
 #include <AK/StringBuilder.h>
 #include <AK/Try.h>
-#include <LibC/sys/arch/i386/regs.h>
+#include <LibC/sys/arch/regs.h>
```

Comment:
This change should have been in the second commit, that moves everyone off of arch/i386/regs.h